templater/templates/periodics.yaml

```diff
@@ -0,0 +1,164 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{ $builderBaseImage := printf "%s:%s" "public.ecr.aws/eks-distro-build-tooling/builder-base" .builderBaseTag -}}
+periodics:
+- name: {{ .prowjobName }}
+  cron: "{{ .cronExpression }}"
+  cluster: "prow-postsubmits-cluster"
+  {{- if .extraRefs }}
+  extra_refs:
+  {{- range .extraRefs }}
+  - org: {{ .Org }}
+    repo: {{ .Repo }}
+    base_ref: {{ .BaseRef }}
+  {{- end }}
+  {{- end }}
+  decoration_config:
+    {{- if .timeout }}
+    timeout: {{ .timeout }}
+    {{- end }}
+    gcs_configuration:
+      bucket: s3://prowdataclusterstack-316434458-prowbucket7c73355c-1n9f9v93wpjcm
+      path_strategy: explicit
+    s3_credentials_secret: s3-credentials
+  {{- if .imageBuild }}
+  labels:
+    image-build: "true"
+    {{- if .localRegistry }}
+    local-registry: "true"
+    {{- end }}
+  {{- end }}
+  spec:
+    serviceaccountName: {{ or .serviceAccountName "periodics-build-account" }}
+    automountServiceAccountToken: false
+    containers:
+    - name: build-container
+      image: {{ or .runtimeImage $builderBaseImage }}
+      command:
+      - bash
+      - -c
+      - >
+{{ .command | indent 8 }}
+      {{- if .envVars }}
+      env:
+      {{- range .envVars }}
+      - name: {{ .Name }}
+        value: "{{ .Value }}"
+      {{- end }}
+      {{- end }}
+      {{- if .imageBuild }}
+      livenessProbe:
+        exec:
+          command:
+          - bash
+          - -c
+          - date +%s > /status/pending
+        periodSeconds: 10
+      {{- end }}
+      {{- if .resources }}
+      resources:
+        {{- if .resources.Requests }}
+        requests:
+          memory: "{{ .resources.Requests.Memory }}"
+          cpu: "{{ .resources.Requests.CPU }}"
+        {{- end }}
+        {{- if .resources.Limits }}
+        limits:
+          memory: "{{ .resources.Limits.Memory }}"
+          cpu: "{{ .resources.Limits.CPU }}"
+        {{- end }}
+      {{- end }}
+      {{- if .volumeMounts }}
+      volumeMounts:
+      {{- range .volumeMounts }}
+      - name: {{ .Name }}
+        mountPath: {{ .MountPath }}
+        readOnly: {{ .ReadOnly }}
+      {{- end }}
+      {{- end }}
+    {{- if .imageBuild }}
+    - name: buildkitd
+      image: moby/buildkit:v0.9.0-rootless
```

Comment:
Lets move this to a constant and pass it in

Reply:
Done